src/Components/SmartComponents/Systems/Systems.js

```diff
@@ -57,11 +57,19 @@ class Systems extends React.Component {
                 {
                     key: 'display_name',
                     title: this.props.intl.formatMessage(messages.systemsColumnHeaderName),
-                    composed: ['facts.os_release', 'display_name']
+                    composed: ['facts.os_release', 'display_name'],
+                    transforms: [pfReactTable.sortable]
                 },
                 {
                     key: 'cve_count',
-                    title: this.props.intl.formatMessage(messages.systemsColumnHeaderCveCount)
+                    title: this.props.intl.formatMessage(messages.systemsColumnHeaderCveCount),
+                    transforms: [pfReactTable.sortable]
+                },
+                {
+                    key: 'updated',
+                    title: this.props.intl.formatMessage(messages.systemsColumnHeaderUpdated),
+                    isTime: true,
```

Comment:
These lines of code has some collisions with system culling commit (fix(Systems): use inventory last seen column). Can you please have a look.